src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -376,6 +376,22 @@ public void Socket_SendToAsyncV4IPEndPointToV4Host_Throws()
             Assert.Equal(SocketError.Fault, args.SocketError);
         }
 
+        // NOTE: on *nix, this API returns ENETUNREACH instead of EFAULT: these platforms
+        //       check the family of the provided socket address before checking its size
+        //       (as long as the socket address is large enough to store an address family).
+        [Fact] // Base case
+        [PlatformSpecific(PlatformID.Unix)]
+        public void Socket_SendToAsyncV4IPEndPointToV4Host_Throws_Unix()
```

Comment:
I think the first `V4` refers to the protocol used by the `IPEndPoint`, not the socket.